src/vue/modules/create-asset-sale/wrappers/asset.record.js

```diff
@@ -0,0 +1,105 @@
+import { ASSET_POLICIES } from '@tokend/js-sdk'
```

Comment:
Be sure that this record doesn't have useless constructor properties or getters. If you create a new record for your new module, you need to define only properties, that your new component uses.

Reply:
ok